clouddriver-google-common/src/main/groovy/com/netflix/spinnaker/clouddriver/googlecommon/deploy/GoogleCommonSafeRetry.groovy

```diff
@@ -83,46 +83,45 @@ abstract class GoogleCommonSafeRetry {
                                             Long retryIntervalBase,
                                             Long jitterMultiplier,
                                             Long maxRetries) {
+    def maxAttempts = Math.max(0, maxRetries) + 1
     SafeRetryState state = new SafeRetryState()
-    try {
-      task?.updateStatus phase, "Attempting $action of $resource..."
-      state.finalResult = operation()
-      state.success = true
-      return state
-    } catch (GoogleJsonResponseException | SocketTimeoutException | SocketException e) {
-      // Don't retry if we don't have to.
-      if (e instanceof GoogleJsonResponseException && e.statusCode in successfulErrorCodes) {
-        state.success = true
-        return state
-      } else if (!isRetryable(e, retryCodes)) {
-        throw e
-      }
-      log.warn "Initial $action of $resource failed, retrying..."
 
-      while (state.tries < maxRetries) {
-        try {
-          def tries = ++state.tries
+    while (state.tries < maxAttempts) {
+      def tries = ++state.tries
+      try {
+        if (tries == 1) {
+          task?.updateStatus phase, "Attempting $action of $resource..."
+        } else {
           // Sleep with exponential backoff based on the number of retries. Add retry jitter with Math.random() to
           // prevent clients syncing up and bursting at regular intervals. Don't wait longer than a minute.
           Long thisIntervalWait = TimeUnit.SECONDS.toMillis(Math.pow(retryIntervalBase, tries) as Integer)
```

Comment:
This looks pre-exisiting and not overly consequential, but it seems like there's an off-by-one error in the exponential backoff.  Because `tries` includes the initial try, the first wait interval will be `retryIntervalBase ^ 2` instead of `retryIntervalBase` as I'd expect for truncated exponential backoff.